lib/resources/majors.tsx

```diff
@@ -2653,6 +2860,350 @@ const bsMolCell: Major = {
+      pathing: 1,
+      fine_requirements: [
+        {
+          description:
+            '<b>4 Math Courses</b> <br />' +
+            'Complete all of the following: <br />' +
+            'AS.110.108 Calculus I (Physical Sciences & Engineering) <br />' +
+            'AS.110.109 Calculus II (Physical Sciences & Engineering) <i>OR</i> AS.110.113 Honors Single Variable Calculus <br />' +
+            'AS.110.202 Calculus III <br />' +
+            'EN.553.291 Linear Algebra and Differential Equations',
+          required_credits: 16,
+          criteria:
+            'AS.110.108[C]^OR^AS.110.109[C]^OR^AS.110.113[C]^OR^AS.110.202[C]^OR^EN.553.291[C]',
+        },
+        {
+          description:
+            '<b>5 Math Courses</b> <br />' +
+            'Complete all of the following: <br />' +
+            'AS.110.108 Calculus I (Physical Sciences & Engineering) <br />' +
+            'AS.110.109 Calculus II (Physical Sciences & Engineering) <i>OR</i> AS.110.113 Honors Single Variable Calculus <br />' +
+            'AS.110.202 Calculus III <br />' +
+            'AS.110.201 Linear Algebra <i>OR</i> AS.110.212 Honors Linear Algebra <br />' +
+            'AS.110.302 Differential Equations and Applications',
+          required_credits: 20,
+          criteria:
+            'AS.110.108[C]^OR^AS.110.109[C]^OR^AS.110.113[C]^OR^AS.110.202[C]^OR^AS.110.201[C]^OR^AS.110.212[C]^OR^AS.110.302[C]',
+        },
+      ],
+    },
+    {
+      name: 'Statistics Elective',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'Select one of the following statistics courses: <br />' +
+        'EN.553.310 Probability & Statistics for the Physical Sciences & Engineering <br />' +
+        'EN.553.311 Probability and Statistics for the Biological Sciences and Engineering <br />' +
+        'EN.560.348 Probability and Statistics for Civil Engineering',
+      criteria: 'EN.553.310[C]^OR^EN.553.311[C]^OR^EN.560.348[C]',
+    },
+    {
+      name: 'Science',
+      required_credits: 13,
+      min_credits_per_course: 1,
+      description:
+        'The student must complete all the required science courses.',
+      criteria:
+        'AS.030.101[C]^OR^EN.171.101[C]^OR^AS.173.111[C]^OR^AS.171.102[C]^OR^AS.173.112[C]',
+      fine_requirements: [
+        {
+          description:
+            '<b>Introductory Chemistry I</b> <br /> AS.030.101 Introductory Chemistry I',
+          required_credits: 3,
+          criteria: 'AS.030.101[C]',
+        },
+        {
+          description:
+            '<b>General Physics I</b> <br /> AS.171.101 General Physics: Physical Science Majors I',
+          required_credits: 4,
+          criteria: 'EN.171.101[C]',
+        },
+        {
+          description:
+            '<b>General Physics Laboratory I</b> <br /> AS.173.111 General Physics Laboratory I',
+          required_credits: 1,
+          criteria: 'AS.173.111[C]',
+        },
```

Comment:
hmm I see...not sure what to do here. Since we decided to stick to degree audit, it might be better to just do that. Also, it is better to exclude information if there is ambiguity about it than include it and then it turning out to be incorrect later...

Reply:
Did we decide to stick to degree audit? I think Olivia raised some concerns

Reply:
For the most part